core/domain/story_validators.py

```diff
@@ -0,0 +1,219 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validators for story models."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.domain import base_model_validators
+from core.domain import story_domain
+from core.domain import story_fetchers
+from core.platform import models
+import python_utils
+
+(
+    base_models, exp_models,
+    story_models, user_models
+) = models.Registry.import_models([
+    models.NAMES.base_model, models.NAMES.exploration,
+    models.NAMES.story, models.NAMES.user
+])
+
+
+class StoryModelValidator(base_model_validators.BaseModelValidator):
```

Comment:
StoryModel also has prerequisite_skill_ids and acquired_skill_ids in each story node, shouldn't that also be validated?